Pod/Classes/APMAlertController.swift

```diff
@@ -5,12 +5,14 @@
 import UIKit
 import SnapKit
 
-@objc public enum APMAlertControllerStyle: Int {
+@objc
+public enum APMAlertControllerStyle: Int {
     case alert
     case actionSheet
 }
 
-@objc open class APMAlertController: UIViewController {
+@objc
+open class APMAlertController: UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/b9c4a952162e21ef3d6579eda097becf88e6343a/Pod/Classes/APMAlertController.swift#L15"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 225 lines</td>
    </tr>
  </tbody>
</table>